src/models/tests/m_rsc_db_tests.erl

```diff
@@ -28,13 +28,15 @@ modify_rsc_test() ->
     ?assertEqual(true, m_rsc:is_a(Id, text, AdminC)),
 
     ?assertEqual(false, m_rsc:p(Id, is_published, AdminC)),
+    ?assertEqual(undefined, m_rsc:p(Id, publication_start, AdminC)),
```

Comment:
@mworrell Any idea why this test fails? It’s green locally.

Reply:
I've changed the initial `rsc` definition in `z_install.erl`. Awaiting Travis now.